pkg/operator/controller/gatewayclass/subscription.go

```diff
@@ -10,13 +10,27 @@ import (
 
 	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
 
+	securityv1 "github.com/openshift/api/security/v1"
 	operatorcontroller "github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
 
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 )
 
+const (
+	// RequiredSCCRestrictedV2 is name of the "restricted-v2" SCC.
+	RequiredSCCRestrictedV2 = "restricted-v2"
+	// WorkloadPartitioningManagementAnnotationKey is the annotation key for
+	// workload partitioning.
+	WorkloadPartitioningManagementAnnotationKey = "target.workload.openshift.io/management"
```

Comment:
We have this defined here too: https://github.com/openshift/cluster-ingress-operator/blob/1410fe02a76e33c00208fb3d9a9f00fa38cb527e/pkg/operator/controller/ingress/deployment.go#L97  Is there a way to combine these?

Reply:
I could put it in `names.go`, or I could define a new file for these consts that really ought to be in some imported package (in this case, probably in openshift/api) but are not.